tools/ci/tcdownload.py

```diff
@@ -0,0 +1,84 @@
+import argparse
+import os
+
+import requests
+
+import github
+
+
+def get_parser():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--ref", action="store", default="master",
```

Comment:
What is this argument? Passing a ref like `gsnedders/exec-webdriver` produces `github.GithubException.GithubException: 422 {u'documentation_url': u'https://developer.github.com/v3/repos/commits/#get-a-single-commit', u'message': u'No commit found for SHA: gsnedders/exec-webdriver'}` here. Does this actually have to be a commit object?

Reply:
https://developer.github.com/v3/repos/statuses/#list-statuses-for-a-specific-ref

> The ref can be a SHA, a branch name, or a tag name

Is that a branch name? I suspect you want `--repo-name=gsnedders/web-platform-tests --ref=exec-webdriver` or something?

Reply:
Oh, I was expecting it to be a local refish.

Given GitHub exposes all commits in the entire network through any repo (given they don't track which commits are reachable from refs in a given repo), it'd be nice to just resolve refs to commit objects locally and pass that up to the GH API, given then it would work more like most other git CLIs?

Reply:
Sure, but it doesn't seem like a necessary feature to land something here. The current implementation doesn't depend on actually having git or anything, which can be useful in some scenarios.

Reply:
We should clarify what the arg is and that it must be something resolvable on GitHub within that repo in the `--help` string at least, IMO.

Reply:
Done.